filebeat/docs/load-balancing.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[load-balancing]]
+== Configuring Load Balancing
+
+//REVIEWERS: Which outputs support load balancing? I only see the loadbalance option described under Logstash and Redis. Are any other outputs supported?
```

Comment:
redis, logstash and elasticsearch.

Note: Kafka internally uses the same load-balancer module, but only for asynchronous error handling. Loadbalancing for kafka is internally solved (a little different) by client library used.